### ISSUE TITLE
Noth the Plaguebringer 3rd Balcony phase

### ISSUE THREAD

**Issue by gemt** [elysium-project/naxxramas_ptr_feedback#62]
Currently all I have found of sources as to what should happen in the third balcony phase is something along the lines of:  "fire and destruction" and "certain death" . Not found any specifics as to what spawns, and how much of it.

If anyone have any information on the third balcony phase, please share.

**Comment by Braindot** [comment]
I believe I found what is to be a 3rd phase video. https://www.youtube.com/watch?v=u_HfJ01uyJw
Unless im being fooled, the video looks to be dating back to Vanilla, despite it being uploaded to youtube in 2012?
I went thru the video twice now and afaik the 3rd balcony phase starts at 09:50 ish in the video.

The first spawns from his POV seem to be a Plagued Guardian and a Plague Champion from the south-west corner. Another Plagued Champion together with a Plagued Construct from the west side and a Plagued Champion from the north-west corner.

I havent seen a single Plagued Construct spawn in our PTR raid, and we let it go to the 3rd phase on purpose on our last fight

Thats 5 spawns from just his side in the video. I believe there's another Plagued Guardian from north ish side of the room and another Plagued Champion from the other side of the room.
